velox/functions/sparksql/aggregates/tests/SumAggregationTest.cpp

```diff
@@ -449,5 +449,17 @@ TEST_F(SumAggregationTest, decimalRangeOverflow) {
       {expected},
       {});
 }
+
+TEST_F(SumAggregationTest, sumFloat) {
+  auto data =
```

Comment:
Does this test fail without the change? Looks like it is same with Presto test which result type is float.

Wonder do we need to backport SumTest to sparksql? https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/aggregates/tests/SumTest.cpp#L78

Reply:
@jinchengchenghh 
Without this patch, the current test will throw an overflow error.
I believe there is no need to test all SumTests again, as the only differences between SumAggregate in Spark SQL and Presto now are the decimal type and the conversion of sum(real) -> double in this PR. The registration of other functions is the same. Of course, if deemed necessary, we can open another PR later to conduct separate tests.